scout/parse/panel.py

```diff
@@ -286,7 +286,8 @@ def parse_panel_app_gene(app_gene, hgnc_map):
 
     hgnc_symbol = app_gene["GeneSymbol"]
     # Returns a set of hgnc ids
-    hgnc_ids = get_correct_ids(hgnc_symbol, hgnc_map)
+    ensembl_ids = app_gene["EnsembleGeneIds"]
+    hgnc_ids = [hgnc_map[ensembl_id] for ensembl_id in ensembl_ids]
```

Comment:
set() if they decide to add more than one build, or some other way of mapping to the same gene..
```suggestion
    hgnc_ids = set([hgnc_map[ensembl_id] for ensembl_id in ensembl_ids])
```